### PR TITLE
Update django-redis version

### PR DESCRIPTION
**Pull request by jburel**
Update version when generating doc

### CODE REVIEW

File: autogen_omero.sh [hunk 1]
```diff
@@ -37,7 +37,7 @@ set +u # PS1 issue
 set -u
 python -m pip install "omero-web>=5.6.0"
 python -m pip install future 'ansible<2.7'
-python -m pip install "django-redis>=4.4,<4.9"
+python -m pip install "django-redis==5.0.0"
```
Comment (sbesson) [inline autogen_omero.sh]: As we work on the Django ecosystem upgrade, for me, this raises the question of whether these Django extra packages `redis`, `cors-pipelines` should not be managed at the `omero-web` level via `extra` dependencies.

Comment (jburel) [reply autogen_omero.sh]: This is certainly a good time to have the discussion

Comment (jburel) [reply autogen_omero.sh]: I think they should be included at the omero-web level, at least the ``redis`` one.
@chris-allan @will-moore @joshmoore what is your feeling?

Comment (joshmoore) [reply autogen_omero.sh]: Certainly no objection for an `extra[]` from my side.

Comment (jburel) [reply autogen_omero.sh]: PR https://github.com/ome/omero-web/pull/359

**Pull request merged**